react-ui/README.md

```diff
@@ -0,0 +1,10 @@
+# Balrog UI
+
+## UI Development
+
+To get started:
+
+1. Run `yarn` to install all dependencies
+2. `yarn start` to start the development server
+
+Once the server is running, navigate to http://localhost:5000 to see the UI.
```

Comment:
From my reading the port should be 9000?

Reply:
Maybe I should just remove this README altogether on import, since the supported way to run is with `docker-compose`. I probably should update the toplevel README, too.